src/express-middleware.js

```diff
@@ -104,9 +110,7 @@ class ExpressMiddleware {
         }
 
         req.metrics = {
-            timer: this.setupOptions.responseTimeHistogram.startTimer({
-                method: req.method
```

Comment:
why isn't this required anymre?

Reply:
@kobik it's because `method` is already included in https://github.com/PayU/prometheus-api-metrics/pull/57/files#diff-50d0b9ad1b1432367f159cd7a82a5921549941cfc939766fc69bd85fda46b0c2R41

Reply:
@kobik it's because `method` is already included in https://github.com/PayU/prometheus-api-metrics/pull/57/files#diff-50d0b9ad1b1432367f159cd7a82a5921549941cfc939766fc69bd85fda46b0c2R41